src/blinding/smokescreen.py

```diff
@@ -196,16 +233,17 @@ def calculate_blinding_factor(self, type="add"):
 
         # update the likelihood with the systematics parameters:
         self.likelihood.update(self.systematics)
+        self._bld_likelihood.update(self.systematics)
```

Comment:
Why do you need a copy of the likelihood and tools when they both presumably read the same sacc file?

Reply:
I need to call two theory vectors: one with a fiducial cosmology and one with a shifted cosmology.

Firecrown safeguards don't allow me to call a theory vector calculation twice.

Reply:
Fixed by [36d45f8](https://github.com/LSSTDESC/Blinding/commit/36d45f80058d837223e7f3aceb7b8704506e8ebb)